frontend/packages/dev-console/src/extensions/add-actions.ts

```diff
@@ -18,6 +18,8 @@ namespace ExtensionProperties {
     url: string;
     /** Optional access review to control visibility / enablement of the action. */
     accessReview?: AccessReviewResourceAttributes[];
+    /** Optional funtion used to show/hide the add action */
+    hide?: (id: string) => boolean;
```

Comment:
No need to supply the id. At declaration time we already know the ID.
```suggestion
    hide?: () => boolean;
```